contents/docs/surveys/branching-logic.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Branching logic
+sidebar: Docs
+showTitle: true
+availability:
+  free: none
+  selfServe: full
+  enterprise: full
+---
+
+It is useful to be able to display survey questions conditionally. For example, you may want to ask certain questions based on the responses received. Branching logic allows you to do this.
+
+## Simple branching
+
+For every question type, you can now define the next step to navigate to after a question is completed. Click on __After this question, go to:__, then choose the next step you want to transition to. The options are:
+
+- Next question
+- Any specific question
+- Confirmation message (if present) or End
+
+After the given question is completed, the chosen next step will be followed.
```

Comment:
I dont understand how this works. Why would you not just proceed to the next logical step e.g. next question / end of survey? What does this enable you to do?

Reply:
Let's say you have: q1, q2, q3, confirmation message.
You are at q1. Depending on how q1 is answered, you will navigate either to q2 or q3.

q2 and q3 should both continue to the confirmation message. For q2 you will need to say this explicitly, otherwise it would continue to q3 by default. Simple branching allows you to specify this ("from q2 always go to the confirmation message")